share/completions/jbake.fish

```diff
@@ -0,0 +1,7 @@
+# Completions for JBake (https://jbake.org/)
+
+complete -c jbake -o b -l bake -f -d "Performs a bake"
```

Comment:
We use the imperative - "Perform a bake", "Print the help message" and so on.

Reply:
I didn't know that, I basically copied the text from the `jbake --help` command itself.